### ISSUE TITLE
Ability to subscribe to a forum category

### ISSUE THREAD

**Issue by hucker75** [BOINC/boinc#4637]
I want to subscribe to something like the forum "Message Boards: News" - https://boinc.berkeley.edu/dev/forum_forum.php?id=22 so I  know when a new project is announced in there.

**Comment by AenBleidd** [comment]
There is an RSS button at the bottom:
![image](https://user-images.githubusercontent.com/3234209/154412351-23743bb6-d1ab-4c23-8cac-61634ad78ab8.jpeg)

**Comment by hucker75** [comment]
Hmmmm..... I don't use RSS.  I suppose I might remember to check every so often.  Had to add the function to my browser.  Would be nicer by email.

**Comment by davidpanderson** [comment]
I agree this would be a good feature (subscribe to a forum, rather than a thread).
Wouldn't be that hard to implement; let's keep it open for a while.
